client/src/views/Search.vue

```diff
@@ -37,11 +37,12 @@ export default defineComponent({
     },
     methods: {
         async onAddKeyword(event: { value: string }) {
-            this.searchTerms.push(event.value);
+            this.searchTerms = this.searchTerms.concat(event.value);
             this.suggestedTerms = await this.keywordService.getSuggestions(this.searchTerms);
         },
 
-        async onRemoveKeyword() {
+        async onRemoveKeyword(event: { index: number; value: string }) {
+            this.searchTerms = this.searchTerms.filter((t, index) => index !== event.index);
```

Comment:
@micahh2 The function property t is not used here. You might want to prefix that with _. Apparently a best practice in JavaScript. Or at least that's what my IDE is telling me 😅

Reply:
That's a great point! I actually normally do that, but I wasn't sure if that was copacetic with our style-guide. After there were no linter errors I figured that this wasn't important.